lib/board.rb

```diff
@@ -34,6 +34,20 @@ def valid_placement?(ship, coordinates)
       false
     end
   end
+
+  def place(ship, coordinates)
+    if valid_placement?(ship, coordinates) #moved this up, if valid_placement?(ship, coordinates) then allow rest of method to run
```

Comment:
refactored here to continue our use of other  local methods to start off our method definition for place(ship, coordinates)